mne/annotations.py

```diff
@@ -977,3 +1024,58 @@ def events_from_annotations(raw, event_id="auto",
                 (list(event_id_.keys()),))
 
     return events, event_id_
+
+
+@verbose
+def annotations_from_events(events, sfreq, event_desc=None, first_samp=0,
+                            orig_time=None, verbose=None):
+    """Convert an event array to an Annotations object.
+
+    Parameters
+    ----------
+    events : ndarray, shape (n_events, 3)
+        The events.
+    sfreq : float
+        Sampling frequency.
+    event_desc : dict | callable | None
+        Events description. Can be:
+
+        - **dict**: map integer event codes (keys) to descriptions (values).
```

Comment:
Do we use ** or ` here?

Reply:
The original annotations.py code used `**` 
I can switch if you want.

Reply:
Nah, if it was like that no need to change.